SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -223,9 +226,98 @@ void USpatialWorkerConnection::ConnectToLocator()
 	});
 }
 
-bool USpatialWorkerConnection::ShouldConnectWithLocator()
+void USpatialWorkerConnection::ConnectToLocator()
+{
+	if (LocatorConfig.WorkerType.IsEmpty())
+	{
+		LocatorConfig.WorkerType = SpatialConstants::ClientWorkerType;
+	}
+
+	if (LocatorConfig.WorkerId.IsEmpty())
+	{
+		LocatorConfig.WorkerId = LocatorConfig.WorkerType + FGuid::NewGuid().ToString();
+	}
+
+	FTCHARToUTF8 PlayerIdentityTokenCStr(*LocatorConfig.PlayerIdentityToken);
+	FTCHARToUTF8 LoginTokenCStr(*LocatorConfig.LoginToken);
+
+	Worker_Alpha_LocatorParameters LocatorParams = {};
+	LocatorParams.player_identity.player_identity_token = PlayerIdentityTokenCStr.Get();
+	LocatorParams.player_identity.login_token = LoginTokenCStr.Get();
+
+	WorkerLocator = Worker_Alpha_Locator_Create(TCHAR_TO_UTF8(*LocatorConfig.LocatorHost), 0, &LocatorParams);
+
+	// TODO: Move creation of connection parameters into a function somehow
+	Worker_ConnectionParameters ConnectionParams = Worker_DefaultConnectionParameters();
+	FTCHARToUTF8 WorkerTypeCStr(*LocatorConfig.WorkerType);
+	ConnectionParams.worker_type = WorkerTypeCStr.Get();
+	ConnectionParams.enable_protocol_logging_at_startup = LocatorConfig.EnableProtocolLoggingAtStartup;
+
+	Worker_ComponentVtable DefaultVtable = {};
+	ConnectionParams.component_vtable_count = 0;
+	ConnectionParams.default_component_vtable = &DefaultVtable;
+
+	ConnectionParams.network.connection_type = LocatorConfig.LinkProtocol;
+	ConnectionParams.network.use_external_ip = LocatorConfig.UseExternalIp;
+
+	FString ProtocolLogDir = FPaths::ConvertRelativePathToFull(FPaths::ProjectLogDir()) + TEXT("protocol-log-");
+#if PLATFORM_WINDOWS == 1 || PLATFORM_XBOXONE == 1
+	ConnectionParams.enable_protocol_logging_at_startup = true;
```

Comment:
Why do we override protocol logging on Windows?
Also, I believe David mentioned in his PR making this part configurable instead of hard-coded.

Reply:
I'm not sure, it looks a little bit like debug/testing code.  It appears that it was already configurable above(line 254).  I'll pull it out.

Reply:
I've pulled this out now.